src/StickyMessages.jl

```diff
@@ -95,7 +95,7 @@ end
 
 function Base.push!(sticky::StickyMessages, message::Pair)
     if !sticky.ansi_codes
-        write(sticky.io, message[2])
+        println(sticky.io, rstrip(message[2]))
```

Comment:
I tweaked how `StickyMessages` is handled in unsupported TTYs a bit.  I think we need to ensure printing newline to be a bit more consistent with how it is handled in supported TTYs.   Otherwise, multi-line sticky messages like nested progress bars are unreadable in such output streams.  I wondered if I should use `chomp` here but I guessed it might be better to print only one newline to be as compact as possible.